apps/google-analytics-4/frontend/src/components/main-app/analytics-app/AnalyticsApp.spec.tsx

```diff
@@ -12,11 +13,6 @@ jest.mock('@contentful/react-apps-toolkit', () => ({
 }));
 
 const mockApi = jest.fn();
-jest.mock('hooks/useApi', () => ({
-  useApi: () => ({
-    runReports: mockApi,
-  }),
-}));
 
 const { findByTestId, getByTestId, getByText, findByText } = screen;
```

Comment:
`findByText` is not being used in this file, it can be removed.

Reply:
nice catch, have removed! 🦅